scripts/css-processing.js

```diff
@@ -21,6 +21,12 @@ const postCSSPlugins = (resourcePath) => {
         require('postcss-inherit')(),
         require('postcss-preset-env')({
             stage: 0,
+            browsers: [
+                'last 2 Chrome versions',
+                'last 2 Firefox versions',
+                'last 3 Safari versions',
```

Comment:
@adixon-adobe updated, and the one thing it "added" was already being shipped, so it doesn't even show up here...good to go?